README.md

```diff
@@ -23,7 +23,7 @@ Use the following table to find the correct plugin version for each SonarQube ve
 
 | SonarQube Version | Plugin Version |
 |-------------------|----------------|
-| 10.6              | 1.22.0         |
+| 10.7 & 10.6       | 1.22.0         |
```

Comment:
```suggestion
| 10.6+                 | 1.22.0         |
```

I prefer this solution and it seems to have been used in earlier instances where same plugin version applied to multiple SQ versions.

Reply:
Thank you for the feedback. I changed it. Later, I realized that I could have used your suggestion.